sql/sql_parse_com_rpc.cc

```diff
@@ -0,0 +1,242 @@
+#include "sql_parse_com_rpc.h"
+#include "srv_session.h"
+
+#ifndef EMBEDDED_LIBRARY
+
+static const char* RpcRoleAttr = "rpc_role";
+static const char* RpcDbAttr = "rpc_db";
```

Comment:
```suggestion
static constexpr char* RpcDbAttr = "rpc_db";
```

Reply:
@percona-ysorokin as I learned in a very hard way, constexpr here results in a constexpr pointer and not string literal :)
```suggestion
static constexpr char RpcDbAttr[] = "rpc_db";
```